src/loader/filesystem_loader.py

```diff
@@ -121,21 +125,22 @@ def get_employees(self) -> list[Employee]:
         return employees
 
     def get_shifts(self) -> list[Shift]:
-        """
-        Actual shifts from timeoffice:
-        return [
-            Shift(1, "Früh", 360, 850),
-            Shift(2, "Spät", 770, 1260),
-            Shift(3, "Nacht", 1220, 390),
-        ]
-        """
-        return [
+        base_shifts = [
             Shift(Shift.EARLY, "Früh", 360, 820),
             Shift(Shift.INTERMEDIATE, "Zwischen", 480, 940),
             Shift(Shift.LATE, "Spät", 805, 1265),
             Shift(Shift.NIGHT, "Nacht", 1250, 375),
         ]
 
+        # Zusätzliche Schichten je nach Case
+        if self._has_special_shifts():
+            base_shifts.append(
+                Shift(4, "Z60", 480, 540)  # 8:00-9:00 Uhr, 60 Min
+            )
+            logging.debug("Special shift Z60 added")
+
+        return base_shifts
+
```

Comment:
We need a more generic approach, we need to get all required shifts from the database, this includes the "plannable" shifts and the non plannable like the Z60, I like the idea of having a property of `plannable` on each shift. Not `plannable` shifts have to be inside the planned shifts.